renderer/containers/ModalStack.js

```diff
@@ -14,41 +14,76 @@ import ChannelDetailModal from 'containers/Channels/ChannelDetailModal'
 import ChannelCreate from 'containers/Channels/ChannelCreate'
 import ReceiveModal from 'containers/Wallet/ReceiveModal'
 import ActivityModal from 'containers/Activity/ActivityModal'
-import Settings from 'containers/Settings'
+import SettingsPage from 'containers/Settings/SettingsPage'
 
 const Container = styled(animated.div)`
   ${ModalOverlayStyles}
 `
 
 const ModalContent = ({ type, closeModal }) => {
-  const doCloseModal = () => closeModal()
   switch (type) {
-    case 'SETTINGS_FORM':
-      return <Settings onSubmit={doCloseModal} />
+    case 'SETTINGS':
+      return (
+        <Modal onClose={closeModal}>
+          <SettingsPage />
+        </Modal>
+      )
 
     case 'AUTOPAY':
-      return <Autopay mx={-4} />
+      return (
+        <Modal onClose={closeModal} p={4}>
```

Comment:
Now that you are using `p={4}` (except settings, but maybe it can be fine-tuned there too) can we move  `<Modal onClose={closeModal} p={4}>` higher and basicall have
```
<Modal onClose={closeModal} p={4}>
   switch
</Modal>
```
?

Reply:
Updated.